index.js

```diff
@@ -62,15 +61,24 @@ module.exports = (options = {}) => {
   const robots = []
 
   function receive (event) {
-    return Promise.all(robots.map(robot => robot.receive(event)))
+    if (event.event && event.payload) {
+      console.log('DEPRECATED: robot.receive({event, payload}) is now robot.receive({id, name, data})')
+      options = {
+        id: 'unknown',
+        name: event.event,
+        data: event.payload
+      }
+    }
+
+    return webhook.receive(event)
```

Comment:
strictly speaking this is a breaking change. Before, you got an array of `undefined` because of the `Promise.all()`, now it just resolves with `undefined`. But I guess in that case I would say we are good. 

Is the method documented anywhere at all?